Sources/XMTPiOS/XMTPLogger.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Logger.swift
+//
+//
+//  Created by Pat Nakajima on 8/28/23.
+//
+
+import Foundation
+import LibXMTP
+import os
+
+class XMTPLogger: FfiLogger {
+	let logger = Logger()
+
+	func log(level: UInt32, levelLabel: String, message: String) {
+		logger.info("libxmtp[\(levelLabel)] - \(message)")
```

Comment:
It would be nice to not always log on info and instead switch on level so
if level is 3 then logger.info

This is how Rust maps

```
Error: 1
Warn: 2
Info: 3
Debug: 4
Trace: 5
```

Reply:
Updated